packages/overlay/overlay-trigger.md

```diff
@@ -123,6 +123,79 @@ The delivery of hover content can be customized via the `placement` attribute. H
 </overlay-trigger>
 ```
 
+## Styling
+
+Content that is thrown into an overlay (e.g. `[slot="*-content"]`) will be moved out of the CSS scope created by its parent `<overlay-trigger>` and other related elements within the same DOM/shadow tree. In this way styling rules that rely on that scope will not travel with the content into its overlay context. In order to ensure that your overlay content is styled as you would like you will need to bind those styles by applying them inline, including a `<style>` element _inside_ of your content element, or by making your content its own custom element.
+
+**Inline Styles**
+
+```html
+<overlay-trigger placement="top-start">
+    <button slot="trigger">Trigger Element</button>
```

Comment:
Just wondering, was the usage of `HTML button` intentional, since rest of the document uses `sp-button`

Reply:
Good question! I was thinking this helped reduce the perception of "magic" here so that readers could focus on the `[slot="click-content"]` element, but maybe you bringing this up shows that I was mistaken and it should be `sp-button` for uniformity?

Reply:
Yeah, `sp-button` would be better in my opinion.